rider-core/src/main/java/com/github/database/rider/core/util/EntityManagerProvider.java

```diff
@@ -53,6 +55,7 @@ public static synchronized EntityManagerProvider instance(String unitName) {
         try {
             instance.init(unitName);
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Exception is logged in the next statement.

Reply:
The log statement depends on user log configuration, otherwise the exception will be omitted. Using printStackTrace we force the exception to be logged, WDYT?

Reply:
Yes, but printStackTrace forces output to console, even if log system is configured to use other storage like file, causing misunderstandings and confusion. Additional confusion for developer exists when to add this printStackTrace and when not.

Reply:
Yes, my main concern is that this exception is omitted by default (if dev dont config the logging)  and the test will run BUT will fail because no database was created. 

I think I'll print the stacktrace only if error log is not configured, WDYT?

Reply:
Probably in this case it is better to throw assertion error rather than continue execution?

Reply:
The problem is that this code is executed before test execution, on a Junit rule and as we can have multiple rules on a test class, if we throw an exception we may invalidate all tests, for example see here: https://github.com/database-rider/database-rider/blob/7d6d39106cb21ea67442f247715ff7bf8e75fee5/rider-core/src/test/java/com/github/database/rider/core/MultipleDataBasesIt.java#L28-L35

Reply:
If rule failed to execute it's main task it must throw exception. Of course, if this is not an expected behavior. :)